plugins/delpoyment_stats/app/controllers/deployment_stats_controller.rb

```
@@ -0,0 +1,40 @@
+      stat.project_name = Project.find_by_id(stat.project).try(:name)
+    end
+
+    render json: @stats
+  end
+
+  def create
+    project = params[:project]
+    stage = params[:stage]
+    status = params[:status]
+
+    result = eval("DeploymentStats.record_deployment('#{project}', '#{stage}', '#{status}')")
```

rubocop (1.68.0)

[convention] The use of `eval` is a serious security risk.

(Security/Eval)